test/cases/ostree-vsphere.sh

```diff
@@ -0,0 +1,884 @@
+#!/bin/bash
+
+# This test scrip covers:
+# 1. build edge-container image, build edge-vsphere image
+# 2. upload image to vsphere datacenter7.0-intel, create edge vm, run test cases against vm.
+# 3. upload image to vsphere datacenter7.0-amd, create edge vm, run test cases against vm.
+# 4. build upgrade edge-container image
+# 5. run rpm-ostree upgrade in all edge vms, then run test cases against upgraded vms.
+# 6. delete all vsphere vms, images
+
+set -euo pipefail
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh none
+
+# Get OS data.
+source /etc/os-release
+ARCH=$(uname -m)
+
+source /usr/libexec/tests/osbuild-composer/shared_lib.sh
+
+# Install sshpass
```

Comment:
We should try to avoiding third party tool. You can use ssh to do the same thing.

Reply:
I will  try to see if govc can do this, thanks.

Reply:
removed sshpass.